docs/UserGuide.md

```diff
@@ -112,6 +114,8 @@ List of `<compulsory arguments>`:
 🚩: Duplicate entry of recurring tasks are **allowed**. 
 
 
+🚩: Duplicate task descriptions are **allowed**. 
+
```

Comment:
Same for this, not required anymore

Reply:
Ohh I took this change when there was a merge conflict haha.

Reply:
ohhh i see, i rephrased it to line 114 instead